src/styles/core/_vars.scss

```diff
@@ -34,6 +34,9 @@ $article-stacked-margin-small: 20px;
 $topic-link-icon-width: 1.294rem;
 $topic-link-icon-spacing: 1rem;
 
+// Icons
+$icon-size: 16px;
```

Comment:
Feel free to ignore this, but I think a name like `$icon-size-default` might be a little safer, because this one kind of implies that all icons will always be this size. That might be the case now (honestly not 100% sure), but I see that there could be a future situation where we have some icons of different sizes in different places in the UI and this variable name might be confusing then.

Reply:
done!